gemfiles/rails_6.1/Gemfile

```
@@ -1,5 +1,5 @@
 source "https://rubygems.org"
 
-gem "rails", "~> 6.1.0"
+gem "rails", "~> 6.1"
```

It is better to restrict at patch level

```suggestion
gem "rails", "~> 6.1.0"
```